src/main/java/com/cozyhome/onlineshop/orderservice/model/OrderDetails.java

```diff
@@ -0,0 +1,34 @@
+package com.cozyhome.onlineshop.orderservice.model;
+
+import com.cozyhome.onlineshop.orderservice.model.enums.PaymentMethod;
+import com.cozyhome.onlineshop.userservice.model.Address;
+import com.cozyhome.onlineshop.userservice.model.User;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.bson.types.ObjectId;
+import org.springframework.data.mongodb.core.mapping.DBRef;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+@Document(collection = "OrderDetails")
+public class OrderDetails {
```

Comment:
maybe will be better to merge order and orderDetails if we use mongodb? in nosql databases normalization is not required